pages/checkout_step_1_page.py

```diff
@@ -16,15 +11,15 @@ def get_input_frame(self, locator):
         return self.driver.find_element(*locator)
 
     def enter_name(self, name):
-        self.get_input_frame(FIRST_NAME_LOCATOR).send_keys(name)
+        self.get_input_frame(CheckoutStep1PageLocators.FIRST_NAME_LOCATOR).send_keys(name)
```

Comment:
To jest okej, aczkolwiek można to ulepszyć. Zamiast podawac za każdym razem nazwę klasy w celu odniesienia się do pojedynczego lokatora można zainicjować obiekt tej klasy w kostruktorze klasy widoku (kompozycja) np. coŚ takiego:
```python
class CheckoutStep1Page(BasePage):
    def __init__(self):
        self.locators = CheckoutStep1PageLocators()
```
I mająć coś takiego w metodach klasy widoku może się do tego odnieść tak:`self.locators.FIRST_NAME_LOCATOR`

I tu oczywiście stary kod konstruktora zostawiasz takim jaki byl mi sie nie chcialo go po prostu przepisywac.